docs/build.rst

```diff
@@ -243,6 +255,15 @@ provided by either PMI2 or PMIx. To enable this support, pass either
 the ``--enable-pmi`` or ``--enable-pmix`` option to configure. Without
 PMI support, a distributed file system accessible to all servers is required.
 
+SPATH
+******
+
+The spath library can be optionally used to normalize relative paths (e.g., ones
+containing ".", "..", and extra or trailing "/") and enable the support of using
+relative paths within an application. To enable, use the ``--with-spath``
+configure option or provide the appropriate ``CPPFLAGS`` and ``LDFLAGS`` at
+configure time.
```

Comment:
Not sure I captured this properly. @adammoody, let me know if you have any suggestions.

Reply:
Yeah, that looks good.  Thanks.